create_conda_software_bundle.sh

```diff
@@ -112,14 +112,13 @@ cd $SB_NAME/bin
 wget https://realearth.ssec.wisc.edu/upload/re_upload || oops "Couldn't download and create re_upload script"
 chmod u+x re_upload || oops "Couldn't make 're_upload' executable"
 
-# Copy SatPy configurations
-mkdir $SB_NAME/etc/satpy || oops "Couldn't create configuration 'etc/satpy' directory"
-cp -r $BASE_P2G_DIR/etc/* $SB_NAME/etc/satpy/ || oops "Couldn't copy configuration 'etc' directory"
-
-conda init bash
-# Restart the shell to enable conda.
-source ~/.bashrc
-conda deactivate
+# Inject environment code into swbundle only.
+for file in `echo *.sh`; do
+    cp "$file" tmp.sh
+    sed "s/# __SWBUNDLE_ENVIRONMENT_INJECTION__/source \$POLAR2GRID_HOME\/bin\/env.sh/g" tmp.sh > "$file"
```

Comment:
Isn't there a `-i` flag for inplace modification?

Reply:
Sed replace in-place is not very universal. There are different requirements between shells and versions, so this is much more maintainable to just explicitly make a temporary file. for example when I do `sed -i "s/# __SWBUNDLE_ENVIRONMENT_INJECTION__/source \$POLAR2GRID_HOME\/bin\/env.sh/g" tmp.sh` I get the error `sed: 1: "tmp.sh": undefined label 'mp.sh'`

Reply:
Ok.